backends/platform/android/android.cpp

```diff
@@ -450,9 +483,7 @@ void OSystem_Android::initBackend() {
 		ConfMan.setInt("gui_scale", 125); // "Large" (see gui/options.cpp and guiBaseValues[])
 	}
 
-
-	ConfMan.registerDefault("savepath", ConfMan.get("path") + "/saves");
-	_savefileManager = new DefaultSaveFileManager(ConfMan.get("savepath"));
+	_savefileManager = new AndroidSaveFileManager(ConfMan.get("path") + "/saves");
```

Comment:
Is it safe to not register a default for key "savepath"? If all checks of this key go through the savefilemanager then it should be. Is this guaranteed?